run-go-generate.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+FILES=$(go list ./...  | grep -v /vendor/)
+exec go generate $FILES
```

Comment:
I think if we wanted to run on all files simply `go generate ./...` would work, but since pre-commit takes care of finding only the modified files, I'm guessing this might be more optimal:

```suggestion
exec go generate "$@"
```

That way `go generate` only runs on modified files. What do you think?